src/utils/init-project/fsh/patient.fsh

```diff
@@ -0,0 +1,6 @@
+// This is a simple example of a FSH file.
+// This file can be renamed, and additional FSH files can be added.
+// SUSHI will look for definitions in any file using the .fsh ending.
+Profile: MyPatient
+Parent: Patient
+* name.family = "Shorty"
```

Comment:
I'm probably being too nit-picky, but this is actually a constraint we would never recommend!  I like our examples / placeholders to follow best practice as much as possible.  So... this could be as simple as `* active = true` or `name 1..* MS` -- but probably not a profile that requires the family name to be Shorty.  Besides, isn't Shorty his given name?

Reply:
Shorty is both Shorty's first and last name, Shorty only needs the one name.